### PR TITLE
testing

### PR DESCRIPTION
**Pull request by mukundarizon**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,5 +9,6 @@
   </head>
   <body>
 	<script src="index.js"></script>
+  <hi>testing</hi>
```
Comment (servicemk) [inline index.html]: make the string red ?

**Review by mukundarizon — COMMENTED** [review]
sure can do

**Review by servicemk — CHANGES_REQUESTED** [review]
lkjhkjh

**Review by servicemk — APPROVED** [review]

**Review by mukundarizon — COMMENTED** [review]
kkjllkjh

**Pull request merged**